lib/src/polymer_base.dart

```diff
@@ -229,6 +229,12 @@ abstract class PolymerBase implements CustomElementProxyMixin {
         .apply([path, convertToJs(value), fromAbove], thisArg: jsElement);
   }
 
+  /// Notify that a value at a path has been changed (without passing the value).
+  void notifyPath1(String path) {
```

Comment:
nonetheless there's a subtle difference. Using this approach those two usages becomes impossible to distinguish:

 -  `notifyPath("somepath")` and 
 - `notifyPath("somepath",null)` 

But potentially they have different meanings. In the first case we don't specify the value and therefore the current value is used, the second we want to notify null).

Should we use some default constant to differentiate those two cases, like this:

    class MissingValue {
      const MissingValue();
    }

  notifyPath(String path,[value = const MissingValue(), fromAbove=false]) { .... }